tox.ini

```diff
@@ -9,13 +9,15 @@ deps =
     {py36,py27,pypy}-unit: coverage
     {py27,pypy}-lint: pylint
     {py27,pypy}-lint: incremental
+    {py27,pypy}-lint: gather
     {py36,py27,pypy}-{func,unit}: Twisted
+    {py36,py27,pypy}-{func,unit}: gather
 commands =
     {py36,py27,pypy}-unit: python -Wall -Wignore::DeprecationWarning -m coverage run -m twisted.trial --temp-directory build/_trial_temp {posargs:ncolony}
     {py36,py27,pypy}-unit: coverage report --include ncolony* --omit */tests/*,*/interfaces*,*/_version* --show-missing --fail-under=100
     py27-lint: pylint --rcfile admin/pylintrc ncolony
     py27-lint: python -m ncolony tests.nitpicker
-    {py36,py27,pypy}-func: python -Werror -W ignore::DeprecationWarning:site -m ncolony tests.functional_test
+    {py36,py27,pypy}-func: python -Werror -W ignore::DeprecationWarning -W ignore::ImportWarning -m ncolony tests.functional_test
```

Comment:
The fact that pkg_resources barfs on its own namespace packages is pretty sad :(

Reply:
Yes :(